.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,18 @@
+name: Create tag
```

Comment:
Yes, I had that in mind as well, but didn't do it because this action actually only tag the repo.
But with the "Start" keyword, I think we can do it.

Reply:
Ok, for consistency, I would also rename the workflow file